packages/xpetra/src/Headers/Xpetra_UseDefaultTypes.hpp

```diff
@@ -54,7 +54,19 @@
 // Define default data types
 typedef double Scalar;
 typedef int LocalOrdinal;
+
+// Choose global ordinal based on enabled underlying libraries
+#ifdef HAVE_XPETRA_TPETRA
+typedef typename Tpetra::Map<>::global_ordinal_type GlobalOrdinal;
```

Comment:
Brian and I talked about this in person today.  I suggested adding `#error` if Epetra and Tpetra are enabled but Tpetra's default ordinal type is not `int`.  That's already an error in Xpetra and it might already catch that case at configure time, but just in case, it would be good to add a test here.